### ISSUE TITLE
TeamViewer Compatibility And Window Focus

### ISSUE THREAD

**Issue by ScottNHa** [Rawns/Reddit-Wallpaper-Changer#42]
Hi devs!

Just want to say I found your program today and absolutely love it already. However, an issue I've run into is that when TeamViewer is launched and the wallpaper is removed, RWC enters a loop of attempting to connect and get wallpapers and the 50 attempts pop up continually shows up.

Is there a way to disable RWC while the system is running a TV connection?

Also, I have found that when the wallpaper changes, it removes focus away from the current window. If I am typing something, it becomes an annoyance to refocus the window. Is it possible to change the wallpaper as truly a background service without losing Window Focus?

**Comment by Rawns** [comment]
Hummm, really not sure about Q1. I don't have access to teamviewer but do have access to something similar, so I'll do some testing and report my findings back.

You can disable RWC by right clicking on the system tray icon (or in the app) and clicking on 'Running' to change it to 'Not running: https://i.imgur.com/jhGO76y.jpg

Do you have the new 'Wallpaper info popup' enabled? If so, I have a feeling that might be grabbing focus from your current active window. If you do, then I'd suggest disabeling it for the moment in the settings to prevent loss of focus.

**Comment by Rawns** [comment]
Right, I've fixed point 3. The info popup will now appear front without grabbing focus. 

Point 1, I've tested with VNC and Splashtop which I do have use of, and RWC continues to works as normal. On VNC connecting, it does remove the background and put the Windows theme to basic (which I believe is standard for increased performance?), but once the next scheduled wallpaper change takes place, the wallpaper is put back even if still connected to VNC. Splashtop seems to leave the theme and wallpaper alone as it's less intensive.

Granted it's not an accurate test as it's not using teamviewer, but I'm sure the connection/running process is roughly the same. I will try and get hold of a copy to test though.
